docker/docker.go

```diff
@@ -357,9 +374,39 @@ func (d *docker) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, e
 
 // GetConfigPolicy returns plugin config policy
 func (d *docker) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	return cpolicy.New(), nil
+	policy := pluginlib.NewConfigPolicy()
```

Comment:
I know it's WIP, I'm warning you are mixing the old library and the new library.

If you want to switch to the new Go lib (which would be great!), you need to do the change everywhere in the code.

Basically, it's removing all those dependency:
```
 	"github.com/intelsdi-x/snap-plugin-collector-docker/client"
 	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
 	"github.com/intelsdi-x/snap/control/plugin"
 	"github.com/intelsdi-x/snap/control/plugin/cpolicy"
 	"github.com/intelsdi-x/snap/core"
```